### DISCUSSION TITLE
Column Selection and Editing

### DISCUSSION THREAD

**Discussion by rrousselot** [dbeaver/dbeaver#13517] [Q&A] [answered]
Please implement the following in the editor.  Notepad++ used as example.

Column Mode & Column Editor
Using Alt + Mouse dragging or Alt + Shift + Arrow keys to switch to column mode:

![image](https://npp-user-manual.org/docs/images/columnMode.gif)

**Comment by ShadelessFox** [comment, marked as answer]
It's already implemented: <kbd>Alt+Shift+A</kbd>
